tests/acceptance/usage-test.js

```diff
@@ -63,30 +63,6 @@ module('Acceptance | usage', function(hooks) {
         assert.dom(selector).exists();
       });
     });
-
-    module('after some time...', function() {
```

Comment:
I think these tests should still exist? why remove them?

Reply:
Yeah, I should have mentioned...

So the first test does not make sense anymore, as we don't have that "fast" notifier anymore. As there are no different polling intervals possible anymore *per component invocation*. See the PRs description.

The second test breaks due to the fact that the notification is now instant, i.e. there is no (artificial) delay anymore (this one: https://github.com/topaxi/ember-service-worker-update-notify/pull/20/files#diff-1612799f82295702bc820f83f8517ba8L75). So the third assertion (after that first 1100ms delay) does not pass anymore. That test was basically only testing that we have that initial delay. Without that delay now, the test would basically be the same as this already existing one: https://github.com/topaxi/ember-service-worker-update-notify/blob/master/tests/acceptance/usage-test.js#L60-L64

What we do not test now, but AFAICT have never really tested, is the further polling, that it correctly recognizes a new sw update after the app is running. But this is due to the `serviceWorkerUpdate()` test helper basically short-cutting the update detection mechanism (as we don't do any real network requests to check for updates in tests).

Reply:
thanks for the thorough explanation!